my-demo-project/model/src/main/java/Teacher.java

```diff
@@ -9,11 +7,22 @@ public Teacher(String name, int salary) {
         this.salary = salary;
     }
 
+    public Teacher() {
+    }
+
```

Comment:
Опять таки добавляем какой-нибудь комментарий

Не забывай в классы, которые используются в коллекциях, вкручивать equals и hashCode

```suggestion
    public Teacher() {
        //Default constructor
    }

```